diff --git a/adm/cmake/tbb.cmake b/adm/cmake/tbb.cmake
index 4e5f724ac2..b0bbd7fc2b 100644
--- a/adm/cmake/tbb.cmake
+++ b/adm/cmake/tbb.cmake
@@ -191,7 +191,7 @@ else ()
     string(TOUPPER "${LIB}" LIB_UPPER)

     # Achive *.so files and directory containing it.
-    get_target_property (TBB_SO_FILE "TBB::${LIB_LOWER}" IMPORTED_LOCATION_RELEASE)
+    get_target_property (TBB_SO_FILE "TBB::${LIB_LOWER}" IMPORTED_LOCATION_NONE)
     # Reserve cache variable for *.so.
     if (NOT DEFINED 3RDPARTY_${LIB_UPPER}_LIBRARY)
       set (3RDPARTY_${LIB_UPPER}_LIBRARY "" CACHE FILEPATH "${LIB_UPPER} library (*.so)")